src/shared/identityUtils.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import fs from 'node:fs';
+import { randomBytes } from 'node:crypto';
+import { DevServerUtils } from './devServerUtils.js';
+
+class LwrConfigFile {
+  public identityToken?: string;
+}
+
+const BYTES_TO_GENERATE = 32;
```

Comment:
I would remove this and instead have `createIdentityToken(byteSize = 32) { ... }`

Reply:
Also usually we should be considering bit size not byte size. I know that for our particular case we want 256-bit = 8-byte size but is it always going to be the case that we want to generate as a factor of 8-bits/1-byte ?

Reply:
I guess since the method is called `createIdentityToken` then it is ok to have it as a factor of 8-bits/1-byte

Reply:
I can add a default parameter and set it to 32. Is the reason you want to parameterize is because you want to be able to control the size of token created? If so then I understand. But also then I see it as a future proofing that's not needed at this time when looking at it with the lens of how often do callers of this method be looking at a fairly large token string in 256-bit.

Another question that I have. Are there times when there are more than 8 bits into a byte?

Reply:
1. We should try to avoid using global variables (even class level variables) when it may not be necessary. I think in this case, passing as a param with default value helps us remove the global variable and also makes the method generic enough to accept different byte sizes.

2. There will always be 8-bits in a byte. My question around the bits/bytes was more in relations to the fact that this method will always generate a token that is a number of bytes long. Will it ever be the case that we would want to call a method to generate a token that is 134-bits long (which is not divisible by 8)? That was my question.

But then again since the method is called `createIdentityToken` (and uses ` randomBytes` to generate) then it is ok that this method always returns a token with a length that is a factor of 8. So you can ignore the comment regarding bit/byte size.